src/main.rs

```diff
@@ -173,7 +174,7 @@ async fn main() -> Result<(), GenericError> {
                     let response = match client
                         .post(format!(
                             "{}/database/deleteGJMessages20.php",
-                            BOOMLINGS_SERVER.as_str()
+                            utils::proxy_list().await.unwrap()
```

Comment:
as i've said before, i really dislike the use of `.unwrap()` and `.expect()` - in this case, i'd prefer `.unwrap_or(BOOMLINGS_SERVER.as_str())`

Reply:
Unwrap_or is already being performed in the function that is being called

Reply:
i can't see where? i can however see the `?` operator which can throw errors and usage of `.unwrap` which can panic

Reply:
function being called is actually unable to fail, so using unwrap is safe here

Reply:
it isn't unable to fail: if it really *is* unable to fail it shouldn't return a `Result`

Reply:
But it to return a result?